app/src/main/java/org/eyeseetea/malariacare/monitor/utils/BaseSurveyMonitor.java

```diff
@@ -0,0 +1,142 @@
+package org.eyeseetea.malariacare.monitor.utils;
+
+import org.eyeseetea.malariacare.database.model.Survey;
+import org.eyeseetea.malariacare.database.model.Value;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by idelcano on 03/08/2016.
+ *
+ * BaseSurveyMonitor contains the common SurveyMonitor methods to each variant
+ */
+public class BaseSurveyMonitor {
+
+    /**
+     * Reference to inner survey
+     */
+    final Survey survey;
```

Comment:
A getter method is not necessary here, but i don't know why the survey is final, I change it to private.